src/js/components/Collapsible/__tests__/__snapshots__/Collapsible-test.js.snap

```diff
@@ -65,7 +65,7 @@ exports[`Collapsible onClick open default 2`] = `
     style="max-height: 0px;"
   >
     <span
-      class="StyledText-sc-1sadyjn-0 hUokoe"
+      class="StyledText-sc-1sadyjn-0 dqKKpZ"
```

Comment:
This change usually means we changed a default behavior (which will be a potential backward compatibility issue). I believe that if you address my comment below on maintaining the default value it will fix the snapshot changes.